torch/_dynamo/variables/dicts.py

```diff
@@ -346,7 +346,13 @@ def getitem_const_raise_exception_if_absent(
     def getitem_const(self, tx: "InstructionTranslator", arg: VariableTracker):
         key = ConstDictVariable._HashableTracker(arg)
         if key not in self.items:
-            unimplemented(f"dict KeyError: {arg.value}")
+            msg = f"Dictionary key {arg.value} not found during tracing"
+            unimplemented_v2(
+                gb_type="dict_key_error",
```

Comment:
The gb_type style we're going for at the moment is something more English-like, so maybe something like "key not found in dict".